public/js/components/App.js

```diff
@@ -60,8 +60,8 @@ const App = React.createClass({
   },
 
   componentDidMount() {
-    this.shortcuts = new KeyShortcuts({ window });
-    this.shortcuts.on("CmdOrCtrl+P", this.toggleSourcesSearch);
+    const shortcuts = setupShortcuts(window);
```

Comment:
oh interesting. I'd like to give react context a try. I'd propose doing it in a separate PR as this should not become a shortcuts PR though.

> In fact, you could just pass down a function 

I'm not sure if that would help because you'd still need to check if shortcuts is available. Also, we would still be passing a prop down the tree.

I'd like to investigate refactoring shortcuts in a later pass